docs: add Jira docs (#385)
casdoor/casdoor-website: docs/integration/java/jira.md

---
title: Jira
description:  Using OIDC protocol as IDP to connect various applications, like Jira
keywords: [OIDC, Jira, IDP]
---

[Casdoor](/docs/basic/server-installation) can use OIDC protocol as IDP to connect various applications. Here we will use [Jira](https://www.atlassian.com/software/jira/guides/getting-started/overview) as an example to show you how to use OIDC to connect to your applications.

The following are some of the names in the configuration:

`CASDOOR_HOSTNAME`: Domain name or IP where Casdoor server is deployed.

`Jira_HOSTNAME`: Domain name or IP where Jira is deployed.

## Step1. Deploy Casdoor and Jira

Firstly, the [Casdoor](/docs/basic/server-installation) and [Jira](https://www.atlassian.com/software/jira/guides/getting-started/overview) should be deployed.

After a successful deployment, you need to ensure:

1. Set Jira URL(Plans -> Administration -> System -> General configuration) to `Jira_HOSTNAME`.
![Jira URL](/img/integration/java/jira/Jira_HOSTNAME.png)
2. Casdoor can be logged in and used normally.

> 3. I use default prod so my CASDOOR_HOSTNAME = 'http://localhost:8000'.When deploy Casdoor in `prod` mode. See [production mode](https://casdoor.org/docs/basic/server-installation#production-mode).

Reviewer: ```suggestion
3. You can set CASDOOR_HOSTNAME = `http://localhost:8000`. When deploy Casdoor in `prod` mode. See [production mode](https://casdoor.org/docs/basic/server-installation#production-mode).
```
